src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image-imac"
+        src="./images/imac.jpeg"
+        alt="imac 27"
+      >
+
+      <h2 class="card__name-of-product">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__code-of-product">Product code: 195434</p>
+
+      <div class="rating card__product-rating">
+        <div class="stars stars--4"
+        >
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="rating__reviews">Reviews: 5</p>
+      </div>
+
+      <div class="price card__price">
+        <p class="price__text">Price:</p>
+
+        <p class="price__value">$2,199</p>
+      </div>
```

Comment:
- The `price' block is not needed, as it is written in the instructions for the task also.
- It's not two paragraph, it's only one, e.x.:

```suggestion
      <p class="card__price">
        <span class="card__price-text">Price:</span>

        <span class="card__price-value">$2,199</span >
      </p>
```